modules/mono/editor/GodotTools/GodotTools.ProjectEditor/default.csproj.template

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Godot.NET.Sdk/%GODOT_NET_SDK%">
+  <PropertyGroup>
+    <RootNamespace>%ROOT_NAMESPACE%</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="!$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net6.0</TargetFramework>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <PublishAot>true</PublishAot>
+    <PublishAotUsingRuntimePack>true</PublishAotUsingRuntimePack>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <ForceFullAOT>true</ForceFullAOT>
+    <SelfContained>true</SelfContained>
+    <GenerateXcodeProject>true</GenerateXcodeProject>
+    <UseNativeAOTRuntime Condition="'$(UseNativeAOTRuntime)' == ''">true</UseNativeAOTRuntime>
+    <Optimize>false</Optimize>
+    <NativeDebugSymbols>true</NativeDebugSymbols>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(RuntimeIdentifier.Contains('iossimulator'))">
+    <LinkerArg Include="-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk" />
```

Comment:
Ideally the path to xcode would come from `xcode-select`, instead of being hardcoded here. However, we can't easily run `xcode-select` before this item group is evaluated (we would have to do it as a target before ilccompile and parse the result, etc), AND ideally the dotnet 8 release won't need these anymore, so this is a temporary workaround. If dotnet 8 release still needs this, then we should revisit this.

Reply:
@ivanpovazan do you know if this is tracked/known somewhere on the dotnet/runtime repo side?

Reply:
We are planning to introduce such changes officially with the support for library mode when targeting iOS with NativeAOT: https://github.com/dotnet/runtime/issues/88737 
Although, that is currently marked with a .NET9 milestone. @MichalStrehovsky should we try to get it in sooner?

Reply:
FYI (and I'll add that comment too upstream, but just so it's here as well), it seems like the workaround is only for the `-lc++` linker argument, and not for the `-isysroot` arg, so it would be super great :heart: :heart: if (at least) the sysroot path inference bit could be fixed for .NET 8

Reply:
I've worked around this issue for now by discovering the Xcode path via `xcrun xcode-select -p`.